pkg/upload/android-proguard.go

```diff
@@ -109,11 +109,7 @@ func ProcessAndroidProguard(apiKey string, applicationId string, appManifestPath
 				for i := range manifestData.Application.MetaData.Name {
 					if manifestData.Application.MetaData.Name[i] == "com.bugsnag.android.BUILD_UUID" {
 						buildUuid = manifestData.Application.MetaData.Value[i]
-						if buildUuid == "" {
-							log.Warn("No BUILD_UUID found in AndroidManifest.xml, defaulting to none")
-						} else {
-							log.Info("Using " + buildUuid + " as build UUID from AndroidManifest.xml")
-						}
+						log.Info("Using " + buildUuid + " as build UUID from AndroidManifest.xml")
```

Comment:
I think we should treat all the fields the same - only log if the value is non-empty. I appreciate that the standard ones (non-Bugsnag) should all be there, but it doesn't hurt to guard them all.